docs/articles/posts/leadership-playbook-change.md

```diff
@@ -0,0 +1,93 @@
+
+When we hear of change, our first step is start processing the message and the immediate impacts. Some people will have questions immediately while others need time to stew on it. 
+
+Because of this, be available to answer questions as they come up (even if you've already answered them before). Be prepared for questions during one-on-ones, after meetings, or whenever they come up. 
+
+Acknowledge the questions and answer them. If you don't know the answer, tell them that you don't know and that you're going to find the answer. 
+
+
+## Giving Space
+
+Don't be surprised that your team exhibits a wide range of emotions for more significant changes. 
+
+For example, if your company is doing layoffs, then it's reasonable for people to be upset, depressed, or mentally checked out. 
+
+When this happens, you have to give people space to process. This doesn't mean isolating them but being aware that they need some time and accommodate accordingly.
+
+A common mistake I see leaders make here is introducing a change, thinking it was low impact, so they start making other changes. In reality, this change had a high impact, and now the team is under pressure to handle the original change and whatever new commitments are coming their way.
+
+Not only does this reduce your odds of success, but this prevents your team from dealing with the changes, which can turn into stress or frustration. If this happens enough times, people will change teams (or even jobs) just to get a change of scenery and be able to process.
+
+## Reducing Change Fatigue
+
+Even though changes are going to happen, do not introduce one change this week, another next week, and then one more two weeks later.
+
+As engineers, we learned that refactoring a codebase should be done in small steps to prevent functionality from breaking. 
+
+While this works great for code, this is terrible advice for humans. 
+
+When frequent changes happen, it becomes difficult to get into a rhythm with the work and the team, reducing the effectiveness of the team.
+
+When we get changes happening like this, it becomes difficult to get into a rhythm with the work and the team. Especially if large changes keep happening every few weeks.
+
+In the current landscape, we're seeing companies go through multiple rounds of layoffs. While this may keep them out of the news (no one reports that a company laid off ten people, even if it's the fourth time it's happened this year), it causes a feeling of dread for the survivors, as now they're thinking when they'll be next.
+
+Instead of having multiple layoffs, if companies had one (albeit larger) layoff, this would allow people to have time to adjust and proceed without as much paranoia.
+
+## Wrapping Up
+
+Navigating change isn't always the easiest, however, if you remember to explain why the change, are transparent about the changes, and are mindful of the impact and recovery of the changes, your chances of keeping the team together will increase.
```

Comment:
I might suggest breaking this up a bit and allowing your key points time to breathe. Maybe something like...

Navigating change is hard – especially when you aren't just going through the change yourself, but also leading others through it at the same time. Try starting here the next time you are going through a change, and see how it goes:

- Explain the 'why' behind the change. 
- Be fully transparent about the changes. 
- Be mindful that everyone processes change differently and may feel a greater impact or need more time to recover from the change.  

If you can start here, your chances of continuing forward without significantly disrupting the team should be far greater.